hawkbit-ddi-resource/src/test/java/org/eclipse/hawkbit/ddi/rest/resource/DdiConfigDataTest.java

```diff
@@ -42,6 +42,12 @@
 @Stories("Config Data Resource")
 public class DdiConfigDataTest extends AbstractRestIntegrationTest {
 
+    /**
+     * Constant class for MediaType HAL with encoding UTF-8. Necessary since
+     * Spring version 4.3.2
+     */
+    private static final String APPLICATION_JSON_HAL_UTF = MediaTypes.HAL_JSON + ";charset=UTF-8";
```

Comment:
This constant is used in a lot of tests, so move it to `AbstractRestIntegrationTest` instead of copy it in every test